### ISSUE TITLE
Axis oriented line segment visualization

### ISSUE THREAD

**Issue by drldcsta** [foxglove/studio#3124]
Currently, our ‘line’ that we use to visualize our path tries to orient itself perpendicular to the camera. I think it would be awesome if foxglove made a new viz type for this that was oriented with the z axis, so that the paths look something like what’s in [this](https://www.youtube.com/watch?v=SbGCIuSyRck&ab_channel=Nuro
) video:

**Comment by jtbandes** [comment]
Can you clarify what "our ‘line’ that we use to visualize our path" is? Is there a screenshot or bag that can be shared, or at least a description of how the visualization is set up in the app right now?

And which path are we talking about from the video? This one that shows the car's trajectory? 
<img width="450" alt="image" src="https://user-images.githubusercontent.com/14237/161167586-f643de66-151a-4f1b-b094-ec012cc278fa.png">

**Comment by AndrewNolte** [comment]
> 

Yes, the visualization right now uses a line type (4) with a width set to the width of the rover. Looking on the other viz types, it may be possible actually to achieve what we want with [triangle list](http://wiki.ros.org/rviz/DisplayTypes/Marker#Triangle_List_.28TRIANGLE_LIST.3D11.29_.5B1.1.2B-.5D) However, it would be nice if this transformation was done on the client-side, or within foxglove itself. Perhaps another viz type could be added.

**Comment by AndrewNolte** [comment]
An example of the foxglove trying to render the line to be perpendicular to the camera:
<img width="462" alt="Screen Shot 2022-04-04 at 10 54 27 AM" src="https://user-images.githubusercontent.com/24322714/161583451-790a5f47-be38-4c8e-9754-e281ee3f4fdd.png">
A more dramatic example:
<img width="505" alt="Screen Shot 2022-04-04 at 10 55 25 AM" src="https://user-images.githubusercontent.com/24322714/161583627-81f8fee8-8d6c-43a0-bdda-ca3c47af6748.png">
The data:
<img width="265" alt="Screen Shot 2022-04-04 at 10 57 35 AM" src="https://user-images.githubusercontent.com/24322714/161584044-bf3c51ef-bd4f-43fa-aca2-005f8bc263ea.png">

**Comment by abhilashchowdhary** [comment]
I believe this is an ask for "first person follow" view  on the selected frame, @AndrewNolte is that correct ?
